### DISCUSSION TITLE
Backref not working at route users/me

### DISCUSSION THREAD

**Discussion by Master-Y0da** [fastapi-users/fastapi-users#590] [Q&A]
I have the following models(using tortoise-orm):

`

    class Auth_Rol_Permiso(models.Model, TimestampMixin):

    id = fields.UUIDField(pk=True)
    permiso = fields.ForeignKeyField('models.Permiso', related_name="auth_permiso", on_delete=fields.CASCADE) 
    user = fields.ForeignKeyField("models.UserModel", related_name="auth_user", on_delete=fields.CASCADE) //base user 

    class Permiso(models.Model):

    id = fields.UUIDField(pk=True)
    nombre = fields.CharField(max_length=100)

With the next pydantic models:

`

    class RolPost(BaseModel):
        nombre: str 

    class UserPermission(BaseModel):
       permiso: RolPost

    class User(fastapi_users_model.BaseUser):
     nombre: str
     auth_user: List[UserPermission] --> If I set auth_user only as a List, it works and returns just the fields from Auth_Rol_Permiso
                                      If I assign List[UserPermission] gives me error auth_user -> 0 -> permiso  field required (type=value_error.missing)` 

Basically I want access the route **users/me** and through backref **auth_user**, get a response like this:
`
 (including nombre from Permiso class, actually it just returns until auth_user array level)

    {
      "nombre" : 'Master Yoda'
      "auth_user" : [ {  "permiso": { "nombre": "Algo" }, "id": 1212 }, {  "permiso": { "nombre": "Algo2" }, "id": 1213 } ] 
     }`

Please give me a hand with this...Thanks for your help.

**Comment by Master-Y0da** [comment]
Any help pls!! @frankie567

**Reply by Master-Y0da** [reply]
This is my current json response at route **users/me** ....what I need is to add data from **permiso_id** relation inside **auth_user**. Any help please!! @frankie567 

[CAPTION](http://i.imgur.com/UjSzse6.png)
